lib/Commands/BuildAllCommand.php

```diff
@@ -73,7 +72,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         assert(is_bool($clearBuildCache));
 
         if ($clearBuildCache) {
-            array_unshift($commands, 'clear-build-cache');
+            $commands = ['clear-build-cache' => [], ...$commands];
```

Comment:
seems to be a bug here, that I solved in this pr when passing the clear-build-cache option.